deploy/spoke/kustomization.yaml

```diff
@@ -30,8 +30,12 @@ resources:
 - ./service_account.yaml
 - ./clusterrole.yaml
 - ./clusterrole_binding.yaml
+- ./clusterrole_addon-management.yaml
+- ./clusterrole_binding_addon-management.yaml
 - ./role.yaml
 - ./role_binding.yaml
+#- ./role_extension-apiserver.yaml
```

Comment:
why comment out these line?

Reply:
these 2 lines moved to Makefile, will remove them.